src/components/presentational/DumbbellChart/DumbbellChart.css

```diff
@@ -0,0 +1,103 @@
+.mdhui-dumbbell-visualization {
+    padding-top: 25px;
+    padding-bottom: 25px;
+    height: 100%;
+}
+
+.mdhui-dumbbell-chart{
+    height: 200px;
+    width: 100%;
+    position: relative;
+}
+
+.mdhui-dumbbell-y-axis{
+    position: absolute;
+    flex-flow: nowrap;
+    display: flex;
+    flex-direction: column;
+    justify-content:flex-end;
+    width: 100%;
+    border-top-width: 1px;
+    border-top-style: solid;
+    border-top-color: #E7EFFF;
```

Comment:
Should be a border color from the global vars...would almost never hardcode colors